app/assets/javascripts/discourse/models/bookmark.js.es6

```diff
@@ -18,6 +26,107 @@ const Bookmark = RestModel.extend({
     return ajax(this.url, {
       type: "DELETE"
     });
+  },
+
+  @discourseComputed("highest_post_number", "url")
+  lastPostUrl(highestPostNumber) {
+    return this.urlForPostNumber(highestPostNumber);
+  },
+
+  // Helper to build a Url with a post number
+  urlForPostNumber(postNumber) {
+    let url = Discourse.getURL(`/t/${this.topic_id}`);
+    if (postNumber && postNumber > 0) {
```

Comment:
```suggestion
    if (postNumber > 0) {
```

In what case would we need to check on postNumber?

Reply:
This is from topic.js model, I will change there too